src/lib/components/ui/Modal/Modal.jsx

```diff
@@ -247,14 +238,7 @@ Modal.defaultProps = {
   id: undefined,
   portalId: null,
   position: 'center',
-  scrollMode: 'body',
-  scrollViewEndShadowStyle: {
-    background: 'radial-gradient(farthest-side at center bottom, rgba(0, 0, 0, 0.16) 0%, rgba(0, 0, 0, 0.06) 40%, rgba(0, 0, 0, 0.02) 85%, rgba(0, 0, 0, 0) 100%)',
-  },
-  scrollViewShadowSize: '16px',
-  scrollViewStartShadowStyle: {
-    background: 'radial-gradient(farthest-side at center top, rgba(0, 0, 0, 0.15) 0%, rgba(0, 0, 0, 0.05) 60%, rgba(0, 0, 0, 0.02) 85%, rgba(0, 0, 0, 0) 100%)',
-  },
+  scroller: (<ScrollView />),
```

Comment:
`<ScrollView />` should contain `scrollViewEndShadowStyle`, `scrollViewShadowSize, `scrollViewStartShadowStyle` defaults from removed `defaultProps`.

Reply:
done